lib/matplotlib/tests/test_backend_qt5.py

```diff
@@ -133,7 +135,7 @@ def test_control_alt():
 def test_modifier_order():
     assert_correct_key(QtCore.Qt.Key_Aacute,
                        (ControlModifier | AltModifier | SuperModifier),
-                       'ctrl+alt+super+' + unichr(225))
+                       'ctrl+alt+super+' + chr(225))
```

Comment:
As far as i know pyqt5 is py3 so I thought this was fine.

On Tue, Oct 6, 2015, 08:37 Michael Droettboom <notifications@github.com>
wrote:

> In lib/matplotlib/tests/test_backend_qt5.py
> <https://github.com/matplotlib/matplotlib/pull/5195#discussion_r41257615>:
>
> > @@ -133,7 +135,7 @@ def test_control_alt():
> >  def test_modifier_order():
> >      assert_correct_key(QtCore.Qt.Key_Aacute,
> >                         (ControlModifier | AltModifier | SuperModifier),
> > -                       'ctrl+alt+super+' + unichr(225))
> > +                       'ctrl+alt+super+' + chr(225))
>
> Maybe this should be six.unichr so it doesn't fail if the default encoding
> is something that can't handle Unicode codepoint 225...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/5195/files#r41257615>.
>